pkg/cmd/server/admin/create_bootstrappolicy_file.go

```diff
@@ -53,6 +55,7 @@ func NewCommandCreateBootstrapPolicyFile(commandName string, fullName string, ou
 
 	flags.StringVar(&options.File, "filename", DefaultPolicyFile, "The policy template file that will be written with roles and bindings.")
 	flags.StringVar(&options.OpenShiftSharedResourcesNamespace, "openshift-namespace", "openshift", "Namespace for shared resources.")
+	flags.MarkDeprecated("openshift-namespace", "this field is no longer supported and using it can lead to undefined behavior")
```

Comment:
We got rid of openshift-infra choice.  I don't remember removing the shared namespace choice.  If openshift-infra was changed, we simply created what needed to be.  If they used a custom shared namespace, we cannot recreate that for them.

@smarterclayton its easier if we remove it, but I'm not sure about breaks.  Opinion?

Reply:
> We got rid of openshift-infra choice. I don't remember removing the shared namespace choice. If openshift-infra was changed, we simply created what needed to be. If they used a custom shared namespace, we cannot recreate that for them.
> 
> @smarterclayton its easier if we remove it, but I'm not sure about breaks. Opinion?

Ansible doesn't allow it to be set.  If a user changes it, I think its up to him to make sure all his roles are ready

Reply:
I think we should remove it.

Reply:
Deprecate in 3.7 and remove in 3.8 seems reasonable to me.  Removing it completely in 3.7 seems a bit sudden, but I have a feeling that no sane person ever messed with this so I can stomp on it if you really want.

Reply:
> Deprecate in 3.7 and remove in 3.8 seems reasonable to me. Removing it completely in 3.7 seems a bit sudden, but I have a feeling that no sane person ever messed with this so I can stomp on it if you really want.

@enj You already have removed it in this pull. You just don't realize that you have.  It no longer functions as it used to.

Reply:
Hm yeah I killed the bootstrapping of it which is what matters.